### PR TITLE
Fixes #119 :Developed Navbar 

### LINKED ISSUE #119: Sections missing from navbar
Sponsors section not added on the navbar and it needs to be developed - generally.

Working on it.

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #119 : Developments in navbar and addition of links

# Things Done:-
### Introduced another dropdown for saving space
  
> @0x48piraj helped me in `header.html`

### CODE REVIEW

**Comment by nzec** [comment]
Preview Link?

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/2012.fossasia.org

**Comment by saarthakchats** [comment]
@duttaditya18

**Comment by duttaditya18** [comment]
@saarthakchats Cannot see any changes.

**Review by akshatnitd — APPROVED** [review]
LGTM !
Please fill the PR template properly from next time.

**Comment by saarthakchats** [comment]
@akshatnitd Filled it. Thanks for letting me know. Will definitely make sure from the next time.

**Review by isuruAb — COMMENTED** [review]
where is the sponsor section ?

**Comment by saarthakchats** [comment]
It's the partners' section. Right below the landing

**Comment by saarthakchats** [comment]
@isuruAb

**Review by isuruAb — APPROVED** [review]

File: _includes/header.html [hunk 1]
```diff
@@ -10,11 +10,17 @@
           <ul class="menu">
             <li><a target="default" class="inner-link" href="{{ baseurl }}./#up">Home</a></li>
             <li><a target="default" class="inner-link" href="{{ baseurl }}./#about">About</a></li>
-            <li><a target="_self" class="inner-link" href="schedule.html">Schedules</a></li>
-            <li><a target="_self" href="workshops.html">Workshops</a></li>
-            <li><a target="_self" href="design.html">Design & Camps</a></li>
+            <li><a target="default" class="inner-link" href="{{ baseurl }}./#partners">Partners</a></li>
+            <li class="has-dropdown">
+            	<a href="workshops.html">Event Elements</a>
```
Comment (anshumanv) [inline _includes/header.html]: Clicking on Event Elements shouldn't take us to `/workshops.html` ideally.

**Comment by saarthakchats** [comment]
@anshumanv Turned it to hashtag, if some relevant page is added, I will add that link later

**Review by nzec — APPROVED** [review]
Cool!

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
@saarthakchats Please change PR title to `Fixes #<issue-number>: <commit-message>` and also change your commit message tto something meaningful

**Comment by arjunpandey2003** [comment]
@saarthakchats there is a problem, you have added designer tab to the event element dropdown but still there is a design tab right next to it. Please take a look
screen shot 2017-12-30 at 1 59 55 pm

**Comment by arjunpandey2003** [comment]
oh sorry my bad they are two different pages
i got confused

**Review by anshumanv — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]

**Comment by saarthakchats** [comment]
@vaibhavsingh97 @anshumanv @saurabhjn76 Pls merge

**Comment by vaibhavsingh97** [comment]
@saarthakchats Please use squash commits and use something meaningful commit message

**Comment by saarthakchats** [comment]
Have squashed, pls merge. @vaibhavsingh97

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**